src/ConnectorRuntime.ts

```diff
@@ -54,6 +55,10 @@ export class ConnectorRuntime extends Runtime<ConnectorRuntimeConfig> {
 
     public readonly infrastructure = new ConnectorInfrastructureRegistry();
 
+    private constructor(connectorConfig: ConnectorRuntimeConfig, loggerFactory: NodeLoggerFactory, private readonly connectorMode: ConnectorMode) {
```

Comment:
Sollte das nicht eher public readonly sein, damit man von anderen Komponenten/Modulen abfragen kann, in welchem Modus man gerade läuft?

Reply:
Falls wir es mal in Modulen brauchen würde ich es in die Module injecten so wie ich es bei der Infrastruktur gemacht habe.